ckan/templates-bs2/package/resource_read.html

```diff
@@ -1,15 +1,15 @@
 {% extends "package/base.html" %}
 
-{% set res = c.resource %}
+{% set res = resource %}
```

Comment:
It's great that we are getting rid of this variables in the `c` object in the templates but AFAICT you are not keeping them in the dataset/resource blueprints. We need to keep them in `c` for at lest a version, otherwise all custom templates that use them will crash. See the approach I suggested here https://github.com/ckan/ckan/pull/3203 (question 1). So basically keep them in `c`, clearly marked for deletion but don't use them in templates.